### DISCUSSION TITLE
what is EoS/EoL date for ACE 7.0.1 and TAO​ 3.0.1?

### DISCUSSION THREAD

**Discussion by sahil3390** [DOCGroup/ACE_TAO#1892] [Q&A] [answered]
Hello,

kindly help in getting the EoS/EoL date for ACE 7.0.1 and TAO​ 3.0.1.

Regards,
Sahil

**Comment by mitza-oci** [comment, marked as answer]
Please see https://github.com/DOCGroup/ACE_TAO/wiki/ACE-and-TAO-Commercial-support for support providers.

**Comment by sahil3390** [comment]
Thanks for sharing support link. 
Can you please also share the EoS/EoL with respective to FOSS ACE 7.0.1 and TAO​ 3.0.1?

**Reply by mitza-oci** [reply]
The commercial support providers, like the company I work for, [Object Computing, Inc.](objectcomputing.com), provide support for FOSS ACE and TAO of many different versions.  In other words, just because support is coming from another entity doesn't keep it from being FOSS.  If you have a support agreement with a provider then that agreement may include an end date.

**Comment by sahil3390** [comment]
Hi team,

We require your input on ACE+TAO(7.0.10) compatibility with RHEL8.6 and
SUSE15 SP3/SP4.
Which version will be compatible with these variants?

On Tue, Jul 26, 2022 at 6:45 PM Adam Mitz ***@***.***> wrote:

> Please see
> https://github.com/DOCGroup/ACE_TAO/wiki/ACE-and-TAO-Commercial-support
> for support providers.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/DOCGroup/ACE_TAO/discussions/1892#discussioncomment-3231098>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALVCJC234YPC6RNE6OF74D3VV7QGHANCNFSM54URQQJQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Thanks And Regards:*

*Sahil Sharma*
*Ph: 09953129412*

**Reply by jwillemsen** [reply]
If you want to have guaranteed compatibility contact one of the commercial support providers like [Remedy IT](https://www.remedy.nl), see https://github.com/DOCGroup/ACE_TAO/wiki/ACE-and-TAO-Commercial-support for all of them. They can validate ACE/TAO with your configuration for your platforms, that is always time consuming, it is a large framework with a lot of features and capabilities.

**Reply by mitza-oci** [reply]
@sahil3390 please use a new discussion for a new question.  This one was answered months ago.
